README.md

```diff
@@ -1,6 +1,23 @@
 # egg.spaces
 
-jvmaynard
-nimejia
-Cynabel
-sydward
+###Featured Developers :D
+
+[jvmaynard](https://github.com/jvmaynard)
+[nimejia](https://github.com/nimejia)
+[Cynabel](https://github.com/cynabel)
+[sydward](https://github.com/radiohYbrid)
+
+###Squadron Leaders <3
+
+[Hypaethral](https://github.com/Hypaethral)
+[dnaf](https://github.com/dnaf)
+
+###Who the heck are we?
```

Comment:
I like these descriptions, so wholesome!  I'm wondering if the "who we are" stuff should be part of the organization description, and we keep the readme specific to the project